FakebookNotifications/FakebookNotifications.DataAccess/Models/Notification.cs

```diff
@@ -0,0 +1,33 @@
+﻿using MongoDB.Bson;
+using MongoDB.Bson.Serialization.Attributes;
+
+namespace FakebookNotifications.DataAccess.Models
+{
+    public class Notification
+    {
+        //mongo object id
+        [BsonId]
+        [BsonRepresentation(BsonType.ObjectId)]
+        public string Id { get; set; }
+
+        //notification type: post/comment/tag
+        [BsonElement("type")]
+        public string Type { get; set; }
```

Comment:
surely we need some kind of target id here too - post id if it's a post, comment id if it's a comment, etc.